src/modules/sessions/controllers/sessions.js

```diff
@@ -1,28 +1,27 @@
+const bcrypt = require('bcrypt')
 const jwt = require('jsonwebtoken')
-const User = require('../models/User')
+const User = require('../../users/models/User')
 
 const userModel = User()
 const jwtSecret = process.env.JWT_SECRET
 
 const create = async (req, res) => {
```

Comment:
```suggestion
const login= async (req, res) => {
```
Esse é o método de login não?

Reply:
É sim
No caso é um 'create session', vi esse padrão numas APIs do job
Vou mudar pra login pra ficar mais óbvio